oid/gen.go

```diff
@@ -10,10 +10,22 @@ import (
 	"log"
 	"os"
 	"os/exec"
+	"strings"
 
 	_ "github.com/lib/pq"
 )
 
+// OID represent a postgres Object Identifier Type.
+type OID struct {
```

Comment:
I could un-export it or rename it, but it helps with only using one query to generate the different structure we need in ``types.go``.